src/types/types.ts

```diff
@@ -713,15 +718,28 @@ export type DeleteAssetMapping3D = AssetMapping3DBase;
 export type EXECUTE = 'EXECUTE';
 
 /**
- * Query schema for asset aggregate endpoint
+ * Query schema for event aggregate endpoint
  */
-export interface EventAggregateQuery {
+export type EventAggregateQuery = EventCountAggregate | EventAdvancedAggregate;
+
+export interface EventCountAggregate {
```

Comment:
Why the change in name here? The `filter` is generic to all aggregates, not only `count` right?

Reply:
Fixed